pkg/nad/masterplugin.go

```diff
@@ -101,6 +101,20 @@ func (plugin *MasterMacVlanPlugin) WithIPAM(ipam *IPAM) *MasterMacVlanPlugin {
 	return plugin
 }
 
+// WithLinkInContainer defines MasterMacVlan plugin using linkInContainer feature.
+func (plugin *MasterMacVlanPlugin) WithLinkInContainer() *MasterMacVlanPlugin {
+	glog.V(100).Infof("Adding linkInContainer configuration to MasterMacVlanPlugin")
+
+	if plugin.masterPlugin == nil {
```

Comment:
Maybe safe to just also check if plugin == nil also

Reply:
According to function NewMasterMacVlanPlugin this condition (plugin == nil) will be always false